ui/src/actions/login.js

```diff
@@ -27,21 +23,6 @@ export function setLoginInfo(loginInfo) {
   };
 }
 
-export function resetLoginInfo() {
-  return setLoginInfo({
-    beamlineName: '',
-    synchrotronName: '',
-    loginType: '',
-    user: '',
-    proposalList: [],
-    selectedProposal: '',
-    selectedProposalID: '',
-    loggedIn: false,
-    rootPath: '',
-    useSSO: false,
-  });
-}
```

Comment:
All we care about is setting `loggedIn` to `false` to trigger the `useEffect` in `App.jsx` to disconnect the sockets. So I'm doing exactly that below in `signOut` by calling `setLoginInfo` directly.